Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -459,6 +475,17 @@ namespace Camera
         UpdateCamera();
     }
 
+    void CameraComponentController::OnViewportDefaultViewChanged(AZ::RPI::ViewPtr view)
+    {
+        if (m_atomCamera != view)
+        {
+            // Note that when disconnected from this bus, this signals that we are not the active view
+            // There is nothing else to do here: leave our view on the viewport context stack, don't need
+            // to update properties. The viewport context system should handle it all!
+            AZ::RPI::ViewportContextNotificationBus::Handler::BusDisconnect();
```

Comment:
Nice fix! Should this reconnect to the bus if `m_atomCamera == view` but we're not already connected to the `ViewportContextNotificationBus`? e.g. if another camera component gets activated and thus pushed on top of us, then deactivated?

Reply:
Yeah it should certainly handle that case!

However we cannot connect to `ViewportContextNotificationBus` if `m_atomCamera == view` because `OnViewportDefaultViewChanged` is an event of `ViewportContextNotificationBus`.

What should happen is we should always listen to the bus to handle precisely the case you've described, and track seperately whether we are the active view (currently `IsActiveView` checks if `ViewportContextNotificationBus` is connected).

Reply:
Addressed in 9a7f6474